libvirt/data_source_libvirt_network.go

```diff
@@ -67,3 +67,85 @@ func resourceLibvirtNetworkDNSHostRead(d *schema.ResourceData, meta interface{})
 
 	return nil
 }
+
+// a libvirt network DNS SRV template datasource
+//
+// Datasource example:
+//
+// data "libvirt_network_dns_srv_template" "etcd_cluster" {
+//   count = "${var.etcd_count}"
+//   service = "etcd-server"
+//   proto = "tcp"
+//   domain = "${discovery_domain}"
+//   target = "${var.cluster_name}-etcd-${count.index}.${discovery_domain}"
+// }
+//
+// resource "libvirt_network" "k8snet" {
+//   ...
+//   dns = [{
+//     srvs = [ "${flatten(data.libvirt_network_dns_srv_template.etcd_cluster.*.rendered)}" ]
+//   }]
+//   ...
+// }
+//
+func datasourceLibvirtNetworkDNSSRVTemplate() *schema.Resource {
+	return &schema.Resource{
+		Read: resourceLibvirtNetworkDNSSRVRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"proto": {
```

Comment:
Why did you used _proto_ when the [libvirt equivalent](https://github.com/libvirt/libvirt-go-xml/blob/6c56ea124a4d6e1099cb264db280dd158da79eac/network.go#L211) is _protocol_?

Reply:
Oops i mxed up names from [DNS SRV RFC 2782](https://www.ietf.org/rfc/rfc2782.txt) and [NetworkDNSSRV](https://github.com/libvirt/libvirt-go-xml/blob/6c56ea124a4d6e1099cb264db280dd158da79eac/network.go#L211)

fixed to match all names to NetworkDNSSRV](https://github.com/libvirt/libvirt-go-xml/blob/6c56ea124a4d6e1099cb264db280dd158da79eac/network.go#L211)